src/v/cluster/types.h

```diff
@@ -2036,10 +2036,13 @@ struct bootstrap_cluster_cmd_data
       const bootstrap_cluster_cmd_data&, const bootstrap_cluster_cmd_data&)
       = default;
 
-    auto serde_fields() { return std::tie(uuid, bootstrap_user_cred); }
+    auto serde_fields() {
+        return std::tie(uuid, bootstrap_user_cred, node_ids_by_uuid);
```

Comment:
should there have been a version bump here?

Reply:
22.3.1 is the first release with this message, so there isn't a compatibility concern. If this PR had landed in 22.3.2 we would've had to.

Reply:
coo